easybuild/easyconfigs/a/Albumentations/Albumentations-1.1.0-foss-2021b.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PythonBundle'
+
+name = 'Albumentations'
+version = '1.1.0'
+
+homepage = 'https://albumentations.ai'
+description = "Albumentations is a Python library for fast and flexible image augmentations"
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('CMake', '3.21.1'),
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('PyYAML', '5.4.1'),
+    ('scikit-image', '0.19.1'),
+    ('scikit-learn', '1.0.1'),
+    ('scikit-build', '0.11.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('opencv-contrib-python-headless', '4.5.5.64', {
```

Comment:
We should replace this by a proper `OpenCV` dependency, and just patch `setup.py` to not require `opencv-python-headless` at all, since our OpenCV installations already come with Python bindings...

You'll need to do the same for `qudida`, which also requires `opencv-python-headless`.

@ItIsI-Orient Can you try that, and then run a couple of tests with the resulting `Albumentations` installation? Should work fine...